build.gradle

```diff
@@ -152,5 +152,11 @@ task signJar(type: SignJar, dependsOn: reobfJar) {
     inputFile = jar.archivePath
     outputFile = jar.archivePath
 }
-
+jar {
+    manifest {
+        attributes([
+                'FMLCorePlugin': "com.mcmoddev.communitymod.CommunityCoremodLoadingPlugin"
+        ])
+    }
+}
```

Comment:
This is not enough to declare the loading plugin. We also need
```gradle
    clientJvmArgs = serverJvmArgs = ["-Dfml.coreMods.load=com.mcmoddev.communitymod.CommunityCoremodLoadingPlugin"]
```
in the `minecraft` block so that the plugin is loaded when generating run configurations and when executing `runClient`/`runServer`.

This must be resolved before merge.